packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -352,7 +353,12 @@ function registerCommands(
 
   const forceFunctionStartCmd = vscode.commands.registerCommand(
     'sfdx.force.function.start',
-    forceFunctionStart
+    forceFunctionContainerlessStartCommand
+  );
```

Comment:
this is where the containerless start function is replacing the previous behavior.